providers/default.rb

```diff
@@ -191,13 +191,26 @@ def chef_backup_dir
   "#{chef_install_dir}.upgrade"
 end
 
+def chef_locked_files_dir
+  "#{chef_install_dir}.locked_files"
+end
+
 # cleanup cruft from *prior* runs
 def cleanup
   if ::File.exist?(chef_backup_dir) # rubocop:disable Style/GuardClause
     converge_by("remove #{chef_backup_dir} from previous chef-client run") do
       FileUtils.rm_rf chef_backup_dir
     end
   end
+  if ::File.exist?(chef_locked_files_dir) # rubocop:disable Style/GuardClause
+    converge_by("remove #{chef_locked_files_dir} from previous chef-client run") do
+      begin
+        FileUtils.rm_r chef_locked_files_dir
+      rescue
+        Chef::Log.warn("Failed to remove locked files dir!")
```

Comment:
I suggest putting the path in here so if you see the warning you know immediately what path to go look at. If you don't put the path in, you should switch to single-quoted strings.
```
Chef::Log.warn("Failed to remove locked files directory #{chef_locked_files_dir}")
```
```
providers/default.rb:210:24: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        Chef::Log.warn("Failed to remove locked files dir!")
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Fixed (do we use that cop?)